projects/observability/src/shared/components/table/data-cell/entity/entity-table-cell-renderer-util.ts

```diff
@@ -27,3 +30,18 @@ export const parseEntityFromTableRow = (cell: Entity | undefined, row: Dictionar
 };
 
 const isInteraction = (neighbor: unknown): neighbor is Interaction => typeof neighbor === 'object';
+
+export const isInactiveEntity = (row: TableRow): boolean | undefined => {
+  const metricAggregations = filterMetricAggregations(row);
+
+  if (metricAggregations.length === 0) {
+    // If an aggregation wasn't fetched, we have no way of knowing if this Entity is inactive.
+    return undefined;
+  }
+
+  return metricAggregations.some(metricAggregation => !isValidMetricAggregation(metricAggregation));
```

Comment:
```suggestion
  return metricAggregations.every(metricAggregation => !isValidMetricAggregation(metricAggregation));
```

Reply:
Hah. That's what I had originally, but figured `some` was sufficient and potentially faster since first match exits. Why do you feel `every` is necessary?

Reply:
Any given metric theoretically could be null (although as far as out of the box metrics, I agree that none come to mind today that would be null when others are present). It seems better to me to be more thorough though, even if it's at the expense of a few microseconds to do an extra handful of iterations.